src/content/docs/en/reference/programmatic-reference.mdx

```diff
@@ -246,4 +279,96 @@ import { sync } from "astro";
 await sync({
   root: "./my-project",
 });
-```
\ No newline at end of file
+```
+
+## `mergeConfig()`
+
+<p>
+
+**Type:** `<T extends AstroConfig | AstroInlineConfig>(config: T, overrides: DeepPartial<T>) => T`
+<Since v="5.4.0" />
+</p>
+
+Imported from `astro/config`, merges a partial Astro configuration on top of an existing, valid, Astro configuration.
+
+`mergeConfig()` accepts an Astro config object and a partial config (any portion of a config, everything is optional), and returns a valid Astro config combining both values such that:
```

Comment:
```suggestion
`mergeConfig()` accepts an Astro config object and a partial config (any set of valid Astro config options), and returns a valid Astro config combining both values such that:
```
I like it! If I tweak it a tiny bit, would this work?

Reply:
Yes